terraform/router/aws/main.tf

```diff
@@ -15,8 +15,10 @@ module "nginx" {
     kubernetes = kubernetes
   }
 
-  namespace = var.namespace
-  rack      = var.name
+  namespace    = var.namespace
+  rack         = var.name
+  replicas_max = var.high_availability ? 2 : 1
+  replicas_min = var.high_availability ? 10 : 1
```

Comment:
Is this the right way round? With HA on, Max is 2 and Min is 10? 😄
What was the default behaviour before adding this?